syft/serde/msgpack/proto.py

```diff
@@ -70,5 +70,7 @@ def proto_type_info(cls):
 
     if type_name in proto_info["TYPES"]:
         return TypeInfo(name=type_name, obj=proto_info["TYPES"][type_name])
+    elif hasattr(cls, "get_msgpack_code"):
```

Comment:
Seems like all classes that inherit from `SyftSerializable` will have this attr from the definition below?

Reply:
Good catch, thanks.